cmd/notation/plugin.go

```diff
@@ -36,6 +44,44 @@ Example - List installed Notation plugins:
 	}
 }
 
+func pluginInstallCommand() *cobra.Command {
+	var force bool
+
+	cmd := &cobra.Command{
+		Use:     "install [flags] <plugin package>",
+		Aliases: []string{"add"},
+		Short:   "Install a plugin",
+		Long: `Install a plugin
+
+		Example - Install a Notation plugin:
+			notation plugin install <plugin package>
+`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return installPlugin(cmd, args, force)
+		},
+	}
+
+	cmd.Flags().BoolVarP(&force, "force", "f", false, "Overwrite existing plugin files without prompting")
```

Comment:
nit: consistent description
```suggestion
	cmd.Flags().BoolVarP(&force, "force", "f", false, "overwrite existing plugin files without prompting")
```